ADAL/src/request/ADAuthenticationRequest+AcquireToken.m

```diff
@@ -147,11 +147,12 @@ - (void)acquireToken:(NSString *)eventName
     }
     
     [[ADTelemetry sharedInstance] startEvent:telemetryRequestId eventName:@"authority_validation"];
-    [[ADInstanceDiscovery sharedInstance] validateAuthority:_context.authority
-                                              requestParams:_requestParams
-                                            completionBlock:^(BOOL validated, ADAuthenticationError *error)
+    
+    ADAuthorityValidation* authorityValidation = [ADAuthorityValidation sharedInstance];
+    [authorityValidation validateAuthority:_context.authority
```

Comment:
Hmmm... authority is in requestParams I believe. I bet we could just pass in that.